consensus/core/src/leader_schedule.rs

```diff
@@ -266,6 +266,11 @@ impl LeaderSwapTable {
         reputation_scores: ReputationScores,
         swap_stake_threshold: u64,
     ) -> Self {
+        #[cfg(msim)]
+        let swap_stake_threshold = 33;
+        #[cfg(not(msim))]
```

Comment:
Are these two lines for `not(msim)` needed?

Reply:
I was trying to fix a linter warning in simtests, but will leave a todo for now

Reply:
If there is a linter warning without these lines, feel free to keep and leave a comment.